src/envo/envo-edit.owl

```diff
@@ -40417,6 +40420,40 @@ During curing process, well-defined and irreversible chemical networks form in t
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105005> "thermoset polymer"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105005> <http://purl.obolibrary.org/obo/CHEBI_60027>)
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105006> (piece of plastic foam)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://www.gesamp.org/publications/guidelines-for-the-monitoring-and-assessment-of-plastic-litter-in-the-ocean>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105006> "A piece of plastic which 1) is partially elastic so that it can deform readily under pressure and 2) has a near-spherical or granular shape."@en)
```

Comment:
>1) is partially elastic so that it can deform readily under pressure and 2) has a near-spherical or granular shape

"...so that it..." should be "...such that it..." 

Also, plastics (which are inelastic) can also deform under pressure. I think this should just be:
"...1) is capable of deforming under pressure..."

You can add a comment that the organisation of the plastic into a foam can give the foam elastic properties (even if the plastic material itself is, by definition, inelastic).


Further this: 
>2) has a near-spherical or granular shape.

Just doesn't seem necessary - there are cubic and other types of plastic foam shapes

Reply:
Hi @pbuttigieg thanks for the comment!

I rethink about the deform part. Even though the GESAMP report define a "foam" as
> Near-spherical or granular particle, which deforms readily under pressure and can be partly elastic, depending on weathering state.

If consider a piece of polystyrene, I think it can fall apart (not deform) under pressure.

I am thinking about to define `piece of plastic foam` as

> A piece of plastic which is composed of foamed plastic.

(and we have defined foamed plastic elsewhere) What do you think?